notebooks/en/rag_with_unstructured_data.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>can you give link to the model repo? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>It is a link already :)</p>